packages/patterns-react/src/patterns/ListSection/ListSection.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { LISTSECTION } from '../../internal/FeatureFlags';
+import ListGroup from './ListGroup';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Section pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.title List section title
+ * @param {string} props.copy List section  short copy to support the title
+ * @param {object} props.listGroup variation of the List section standard, standard with jump link and standard with card link
+ * @returns {object} JSX Object
+ */
+const ListSection = ({ title, copy, listGroup }) =>
+  featureFlag(
+    LISTSECTION,
+    <section
+      data-autoid={`${stablePrefix}--listsection`}
+      className={classNames(`${prefix}--listsection`)}>
+      <div className={`${prefix}--listsection__container`}>
+        <div className={`${prefix}--listsection__row`}>
+          <div className={`${prefix}--listsection__col`}>
+            <h1 className={`${prefix}--listsection__title`}>{title}</h1>
```

Comment:
Is this supposed to be the main item on the page? Just wondering about the `h1` here...

Reply:
we can default to div. h1 would be fine for SEO if pattern is in it's own section. I am not sure how this pattern will be nested within page. Do you think div is fine ?

Reply:
It certainly makes sense for it to be a head tag of some kind though. But if say for example a leadspace is above it with already a H1, that would mess up the semantics of the page.

Reply:
I think we should use h3 to be safe.

Reply:
Changed title tag to h3 by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/398454ef94093019c0b5ad82376816b45195d2e7)